docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -1136,3 +1136,6 @@ connector.
     * - ``delta.projection-pushdown-enabled``
```

Comment:
Use a verb in the commit message

The following resource is quite a good read https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages